src/shared/utils/browser/clear-auth-cookie.ts

```diff
@@ -0,0 +1,12 @@
+import * as cookie from "cookie";
+import { authCookieName } from "shared/config";
+
+export default function clearAuthCookie() {
+  document.cookie = cookie.serialize(authCookieName, "", {
+    expires: new Date("Thu, 01 Jan 1970 00:00:01 GMT"),
```

Comment:
Adding `expires` attribute with value way in the past, which seems to be a common pattern in other code bases.